catalog/spatial/registry/registry-report-viewer/src/main/resources/templates/report.hbt

```diff
@@ -0,0 +1,195 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta content="text/html; charset=windows-1252" http-equiv="content-type" />
+    <!-- styles -->
+    <link href="/services/registries/assets/css/bootstrap.min.css" rel="stylesheet" />
+    <style type="text/css">
+        body {
```

Comment:
Does our formatter get that or should I use an external formatter?
I've been using the atom plugin beautify.